
From: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>,
      Frank Sorenson <frank@tuxrocks.com>

This patch reorders two lines to check a variable for NULL before using 
the variable.

Signed-off-by: Frank Sorenson <frank@tuxrocks.com>
Signed-off-by: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/um/drivers/line.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN arch/um/drivers/line.c~uml-avoid-null-dereference-in-linec arch/um/drivers/line.c
--- 25/arch/um/drivers/line.c~uml-avoid-null-dereference-in-linec	Thu Jan 13 15:27:03 2005
+++ 25-akpm/arch/um/drivers/line.c	Thu Jan 13 15:27:03 2005
@@ -593,8 +593,8 @@ irqreturn_t winch_interrupt(int irq, voi
 		}
 	}
 	tty  = winch->tty;
-	line = tty->driver_data;
 	if (tty != NULL) {
+		line = tty->driver_data;
 		chan_window_size(&line->chan_list,
 				 &tty->winsize.ws_row, 
 				 &tty->winsize.ws_col);
_
